CHANGELOG.md

```diff
@@ -9,105 +9,106 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
-- [#140](https://github.com/equinor/webviz-core-components/pull/140) - Improved styling of the `Select` component.
-- [#145](https://github.com/equinor/webviz-core-components/pull/145) - Added wrapper components for typically used Dash components (Dropdown, Slider, etc) with additional styling.
-- [#148](https://github.com/equinor/webviz-core-components/pull/148) - Changed default value of `numSecondsUntilSuggestionsAreShown` to 0.5 in `SmartNodeSelector` component
+-   [#140](https://github.com/equinor/webviz-core-components/pull/140) - Improved styling of the `Select` component.
+-   [#145](https://github.com/equinor/webviz-core-components/pull/145) - Added wrapper components for typically used Dash components (Dropdown, Slider, etc) with additional styling.
+-   [#148](https://github.com/equinor/webviz-core-components/pull/148) - Changed default value of `numSecondsUntilSuggestionsAreShown` to 0.5 in `SmartNodeSelector` component.
+-   [#151](https://github.com/equinor/webviz-core-components/pull/151) - `SmartNodeSelector`: Changes to `data` and `delimiter` props are considered now and cause the component to update.
```

Comment:
There were quite some changes to this file. Change to only adding the 151-entry, and instead consider if we want to enforce further style rules on `CHANGELOG.md` in separate issue/PR?